packages/infra/lib/api-stack/api-service.ts

```diff
@@ -236,14 +242,16 @@ export function createAPIService({
           }),
         },
       },
-      memoryLimitMiB: isProd(props.config) ? 4096 : 2048,
       healthCheckGracePeriod: Duration.seconds(60),
       protocol: ApplicationProtocol.HTTP,
       listenerPort,
       publicLoadBalancer: false,
-      idleTimeout: Duration.minutes(10),
+      idleTimeout: loadBalancerIdleTimeout,
     }
   );
+  // https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html
+  // fargateService.taskDefinition.defaultContainer?.addUlimits({ ... });
```

Comment:
Added for reference, should we need to set this - took me some time and then realized the defaults are pretty good ([~65,000 file descriptors](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#:~:text=On%20Fargate%2C%20the%20default%20nofile%20soft%20limit%20is%2065535%20and%20hard%20limit%20is%2065535)).